cookbook/security/access_control.rst

```diff
@@ -54,12 +54,10 @@ Take the following ``access_control`` entries as an example:
 
             <config>
                 <!-- ... -->
-                <access-control>
-                    <rule path="^/admin" role="ROLE_USER_IP" ip="127.0.0.1" />
-                    <rule path="^/admin" role="ROLE_USER_HOST" host="symfony\.com$" />
-                    <rule path="^/admin" role="ROLE_USER_METHOD" method="POST, PUT" />
-                    <rule path="^/admin" role="ROLE_USER" />
-                </access-control>
+                <rule path="^/admin" roles="ROLE_USER_IP" ip="127.0.0.1" />
+                <rule path="^/admin" roles="ROLE_USER_HOST" host="symfony\.com$" />
+                <rule path="^/admin" roles="ROLE_USER_METHOD" methods="POST, PUT" />
```

Comment:
`method` is currently not treated as the singular form of `methods` (see symfony/symfony#15202). However, as we have more than one method we must use `methods` nonetheless. Otherwise, the string value would not be split into an array.